daemon/daemon_unix.go

```diff
@@ -375,6 +390,13 @@ func verifyPlatformContainerSettings(daemon *Daemon, hostConfig *containertypes.
 		warnings = append(warnings, "IPv4 forwarding is disabled. Networking will not work.")
 		logrus.Warnf("IPv4 forwarding is disabled. Networking will not work")
 	}
+	if hostConfig.CgroupParent != "" && daemon.usingSystemd() {
+		// CgroupParent for systemd cgroup should be named as "xxx.slice"
+		length := len(hostConfig.CgroupParent)
+		if length <= 6 || hostConfig.CgroupParent[length-6:] != ".slice" {
```

Comment:
Thanks for your tip, I'll update, but what do you mean about this should only apply for absolute paths? Do you mean `hostConfig.CgroupParent` for systemd cgroup should be absolute path?